pycoin/coins/groestlcoin/hash.py

```diff
@@ -0,0 +1,14 @@
+import hashlib
+
+import groestlcoin_hash
```

Comment:
Something like
```suggestion
try:
    from groestlcoin_hash import getHash
except ImportError:
    def getHash(data, size):
        t = "to use groestlcoin, 'pip install goestlcoin'"
        print(t)
        raise ImportError(t)
```

Reply:
This will defer the exception until when the missing function is used.

This is not ideal, but it'll do for now. Eventually we can move to a child package pycoin[groestlcoin] which can include the C libraries.